policies/coding-style.md

```diff
@@ -214,13 +217,22 @@ operators or after the prefix increment and decrement unary operators:
     --bar
 ```
 
-Do not put a space around the '.' and "->" structure member operators:
+Do not put a space around the `.` and `->` structure member operators:
 
 ```c
     foo.bar
     foo->bar
 ```
 
+* Do not use multiple consecutive spaces
```

Comment:
or in comments.  Two spaces after a full stop is pretty widespread.

Reply:
Yep, added.
In fact, `check-format.pl` has a special treatment of two spaces after periods.
And at least in specially formatted comments (starting with `/*-`), multiple spaces are allowed anywhere.